tests/download/test_views.py

```diff
@@ -69,7 +69,7 @@ def test_document_download_with_filename(client, store):
     assert response.get_data() == b'PDF document contents'
     assert dict(response.headers) == {
         'Cache-Control': mock.ANY,
-        'Expires': mock.ANY,
+        'Date': mock.ANY,
```

Comment:
Interesting, I wonder why the switch of the headers from `expires` to `date`.

Reply:
I couldn't find a satisfactory answer for that.  The flask documentation didn't indicate changes to these properties.  But the HTTP spec seems to suggest you don't use both the headers `Cache-control` AND `Expires`, but rather, one or the other.